test/helpers/docker.go

```diff
@@ -102,15 +101,7 @@ func (s *SSHMeta) NetworkGet(name string) *CmdRes {
 }
 
 func (s *SSHMeta) execCmd(cmd string) *CmdRes {
-	stdout := new(bytes.Buffer)
-	stderr := new(bytes.Buffer)
-	exit := s.ExecWithSudo(cmd, stdout, stderr)
-	return &CmdRes{
-		cmd:    cmd,
-		stdout: stdout,
-		stderr: stderr,
-		exit:   exit,
-	}
+	return s.ExecWithSudo(cmd)
```

Comment:
This function doesn't have a purpose - it's not exported, but calls a function that is exported. Replace calls to `execCmd` with `ExecWithSudo` and remove this function.